### ISSUE TITLE
Merge master to develop

### ISSUE THREAD

**Issue by warpech** [StarcounterSamples/Chatter#21]
@diliger, you made code reformating changes on `master` branch, whereas it should be done on `develop` branch (and as a Pull Request). 

Could you please merge `master` to `develop`.

Then please check if `develop` builds with Starcounter stable (2.2.1.3234). If yes, then please merge `develop` to `master` to make the branches even. 

From now on, development is always on feature branches started from `develop`.

**Comment by diliger** [comment]
Merged.
Tested with Starcounter stable (2.2.1.3234).
Works fine.

During testing fixed issue #20 

But noticed some strange behaviour of the `Session.ScheduleTask` function,
if I set `waitForCompletion = true` execution stops on the line and never passes to the next line
even if i put nothing into the task function:
![image](https://cloud.githubusercontent.com/assets/15857369/21468724/86af6230-ca34-11e6-8e50-8f5b74ab932a.png)
